qlkube/src/watch.js

```diff
@@ -63,13 +65,19 @@ function kwatch(api, label) {
       api,
       { allowWatchBookmarks: false },
       (type, apiObj, watchObj) => {
+        graphqlLogger(`[i] Publish event on ${label} label`);
         publishEvent(label, {
           apiObj,
           type,
         });
       },
       err => {
-        console.error(err);
+        if (err.message === 'Not Found') {
```

Comment:
Using error texts to distinguish error types might not be a great deal. Try to see fi there's another field inside which can be more "robust". If this is an http response error there should be a `code` property or something alike, which in this case should be 404. Otherwise if it's a special class (like, SomethingNotFoundError) you could do `if (err instanceof SomethingNotFoundError) ...`